backend/app/tests/services/llm/providers/STTproviders/test_STT_SarvamProvider.py

```
@@ -0,0 +1,193 @@
+            input={"type": "text", "content": {"value": "Transcription request"}}
+        )
+
+        # 4. Execution
+        print("Executing STT with SarvamAIProvider...")
+        # For STT, resolved_input needs to be the file path
+        result, error = instance.execute(completion_config=test_config, query=test_query, resolved_input=temp_audio_file_path)
+
+        if error:
+            print(f"Error: {error}")
+        else:
+            print(f"\n--- SarvamAI STT Result ---")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)